From 7fe398fc3e2ea309177b0f7f30b2e9be13258008 Mon Sep 17 00:00:00 2001
From: Wook Song <wook16.song@samsung.com>
Date: Fri, 29 Apr 2022 10:01:16 +0900
Subject: [PATCH 1/3] Local: Tests: Skip the SSAT-based test cases that failed
 in macOS

This patch skips the following SSAT-based test cases:
- nnstreamer_repo_dynamicity (hanging)
- nnstreamer_grpc (failure)
- nnstreamer_protobuf (failure)
- nnstreamer_flatbuf (failure)
- nnstreamer_flexbuf (failure)

Signed-off-by: Wook Song <wook16.song@samsung.com>
---
 tests/nnstreamer_flatbuf/runTest.sh         | 131 -----------------
 tests/nnstreamer_flexbuf/runTest.sh         | 102 -------------
 tests/nnstreamer_grpc/runTest.sh            | 154 --------------------
 tests/nnstreamer_protobuf/runTest.sh        | 131 -----------------
 tests/nnstreamer_query/runTest.sh           |  52 -------
 tests/nnstreamer_repo_dynamicity/runTest.sh |  57 --------
 6 files changed, 627 deletions(-)
 delete mode 100755 tests/nnstreamer_flatbuf/runTest.sh
 delete mode 100755 tests/nnstreamer_flexbuf/runTest.sh
 delete mode 100755 tests/nnstreamer_grpc/runTest.sh
 delete mode 100755 tests/nnstreamer_protobuf/runTest.sh
 delete mode 100644 tests/nnstreamer_query/runTest.sh
 delete mode 100644 tests/nnstreamer_repo_dynamicity/runTest.sh

diff --git a/tests/nnstreamer_flatbuf/runTest.sh b/tests/nnstreamer_flatbuf/runTest.sh
deleted file mode 100755
index 02ec2ade..00000000
--- a/tests/nnstreamer_flatbuf/runTest.sh
+++ /dev/null
@@ -1,131 +0,0 @@
-#!/usr/bin/env bash
-##
-## SPDX-License-Identifier: LGPL-2.1-only
-##
-## @file runTest.sh
-## @author Gichan Jang <gichan2.jang@samsung.com>
-## @date May 25 2020
-## @brief SSAT Test Cases for flatbuffers subplugin of tensor converter and decoder
-## @details After decoding the tensor into flatbuffer, convert it to tensor(s) again to check if it matches the original
-##
-if [[ "$SSATAPILOADED" != "1" ]]; then
-    SILENT=0
-    INDEPENDENT=1
-    search="ssat-api.sh"
-    source $search
-    printf "${Blue}Independent Mode${NC}"
-fi
-
-# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
-testInit $1
-
-if [ "$SKIPGEN" == "YES" ]; then
-    echo "Test Case Generation Skipped"
-    sopath=$2
-else
-    echo "Test Case Generation Started"
-    python ../nnstreamer_converter/generateGoldenTestResult.py 9
-    python3 ../nnstreamer_merge/generateTest.py
-    sopath=$1
-fi
-convertBMP2PNG
-
-PATH_TO_PLUGIN="../../build"
-
-##
-## @brief Execute gstreamer pipeline and compare the output of the pipeline
-## @param $1 Colorspace
-## @param $2 Width
-## @param $3 Height
-## @param $4 Test Case Number
-function do_test() {
-    gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=3 pattern=13 ! video/x-raw,format=${1},width=${2},height=${3},framerate=5/1 ! \
-    tee name=t ! queue ! multifilesink location=\"raw_${1}_${2}x${3}_%1d.log\"
-    t. ! queue ! tensor_converter ! tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! multifilesink location=\"flatb_${1}_${2}x${3}_%1d.log\" sync=true" ${4} 0 0 $PERFORMANCE
-
-    callCompareTest raw_${1}_${2}x${3}_0.log flatb_${1}_${2}x${3}_0.log "${4}-1" "Flatbuf conversion test ${4}-1" 1 0
-    callCompareTest raw_${1}_${2}x${3}_1.log flatb_${1}_${2}x${3}_1.log "${4}-2" "Flatbuf conversion test ${4}-2" 1 0
-    callCompareTest raw_${1}_${2}x${3}_2.log flatb_${1}_${2}x${3}_2.log "${4}-3" "Flatbuf conversion test ${4}-3" 1 0
-}
-# The width and height of video should be multiple of 4
-do_test BGRx 320 240 1-1
-do_test RGB 320 240 1-2
-do_test GRAY8 320 240 1-3
-
-# audio format S16LE, 8k sample rate, samples per buffer 8000
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=S16LE,rate=8000 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! tensor_decoder mode=flatbuf ! \
-        other/flatbuf-tensor ! tensor_converter ! filesink location=\"test.audio8k.s16le.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio8k.s16le.origin.log\" sync=true" 2-1 0 0 $PERFORMANCE
-callCompareTest test.audio8k.s16le.origin.log test.audio8k.s16le.log 2-2 "Audio8k-s16le Golden Test" 0 0
-
-# audio format U8, 16k sample rate, samples per buffer 8000
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=U8,rate=16000 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! tensor_decoder mode=flatbuf ! \
-        other/flatbuf-tensor ! tensor_converter ! filesink location=\"test.audio16k.u8.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio16k.u8.origin.log\" sync=true" 2-3 0 0 $PERFORMANCE
-callCompareTest test.audio16k.u8.origin.log test.audio16k.u8.log 2-4 "Audio16k-u8 Golden Test" 0 0
-
-# audio format U16LE, 16k sample rate, 2 channels, samples per buffer 8000
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=U16LE,rate=16000,channels=2 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! tensor_decoder mode=flatbuf ! \
-        other/flatbuf-tensor ! tensor_converter ! filesink location=\"test.audio16k2c.u16le.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio16k2c.u16le.origin.log\" sync=true" 2-5 0 0 $PERFORMANCE
-callCompareTest test.audio16k2c.u16le.origin.log test.audio16k2c.u16le.log 2-6 "Audio16k2c-u16le Golden Test" 0 0
-
-# tensor merge test (The output is always in the format of other/tensor)
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN}  tensor_merge name=merge mode=linear option=2 silent=true sync-mode=basepad sync-option=0:0 ! multifilesink location=testsynch08_%1d.log \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)10/1\" ! pngdec ! tensor_converter !  \
-        tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! merge.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)30/1\" ! pngdec ! tensor_converter ! \
-        tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! merge.sink_1" 3 0 0 $PERFORMANCE
-callCompareTest testsynch08_0.golden testsynch08_0.log 3-1 "Tensor merge Compare 3-1" 1 0
-callCompareTest testsynch08_1.golden testsynch08_1.log 3-2 "Tensor merge Compare 3-2" 1 0
-callCompareTest testsynch08_2.golden testsynch08_2.log 3-3 "Tensor merge Compare 3-3" 1 0
-callCompareTest testsynch08_3.golden testsynch08_3.log 3-4 "Tensor merge Compare 3-4" 1 0
-
-# tensor mux test (The output is always in the format of other/tensors)
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_mux name=tensors_mux sync-mode=basepad sync-option=1:50000000 ! multifilesink location=testsynch19_%1d.log \
-    tensor_mux name=tensor_mux0  sync-mode=slowest ! tensors_mux.sink_0 \
-    tensor_mux name=tensor_mux1  sync-mode=slowest ! tensors_mux.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)10/1\" ! pngdec ! \
-        tensor_converter ! tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! tensor_mux0.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! \
-        tensor_converter ! tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! tensor_mux0.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)30/1\" ! pngdec ! \
-        tensor_converter ! tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! tensor_mux1.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! \
-        tensor_converter ! tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! tensor_mux1.sink_1" 4 0 0 $PERFORMANCE
-callCompareTest testsynch19_0.golden testsynch19_0.log 4-1 "Tensor mux Compare 4-1" 1 0
-callCompareTest testsynch19_1.golden testsynch19_1.log 4-2 "Tensor mux Compare 4-2" 1 0
-callCompareTest testsynch19_2.golden testsynch19_2.log 4-3 "Tensor mux Compare 4-3" 1 0
-callCompareTest testsynch19_3.golden testsynch19_3.log 4-4 "Tensor mux Compare 4-4" 1 0
-callCompareTest testsynch19_4.golden testsynch19_4.log 4-5 "Tensor mux Compare 4-5" 1 0
-
-# test other/tensors
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_mux name=tensors_mux sync-mode=basepad sync-option=1:50000000 ! \
-    tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! multifilesink location=testsynch19_5_%1d.log \
-    tensor_mux name=tensor_mux0  sync-mode=slowest ! tensors_mux.sink_0 \
-    tensor_mux name=tensor_mux1  sync-mode=slowest ! tensors_mux.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)10/1\" ! pngdec ! tensor_converter ! tensor_mux0.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! tensor_converter ! tensor_mux0.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)30/1\" ! pngdec ! tensor_converter ! tensor_mux1.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! tensor_converter ! tensor_mux1.sink_1" 5 0 0 $PERFORMANCE
-callCompareTest testsynch19_0.golden testsynch19_5_0.log 5-1 "Tensor mux Compare 5-1" 1 0
-callCompareTest testsynch19_1.golden testsynch19_5_1.log 5-2 "Tensor mux Compare 5-2" 1 0
-callCompareTest testsynch19_2.golden testsynch19_5_2.log 5-3 "Tensor mux Compare 5-3" 1 0
-callCompareTest testsynch19_3.golden testsynch19_5_3.log 5-4 "Tensor mux Compare 5-4" 1 0
-callCompareTest testsynch19_4.golden testsynch19_5_4.log 5-5 "Tensor mux Compare 5-5" 1 0
-
-# Consecutive converting test
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=S16LE,rate=8000 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! \
-    tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! \
-    tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! \
-    tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! filesink location=\"test.consecutive.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio8k.s16le.origin.log\" sync=true" 6 0 0 $PERFORMANCE
-callCompareTest test.audio8k.s16le.origin.log test.consecutive.log 6-1 "Consecutive converting test" 0 0
-
-rm *.log *.bmp *.png *.golden *.raw *.dat
-
-report
diff --git a/tests/nnstreamer_flexbuf/runTest.sh b/tests/nnstreamer_flexbuf/runTest.sh
deleted file mode 100755
index af7befab..00000000
--- a/tests/nnstreamer_flexbuf/runTest.sh
+++ /dev/null
@@ -1,102 +0,0 @@
-#!/usr/bin/env bash
-##
-## SPDX-License-Identifier: LGPL-2.1-only
-##
-## @file runTest.sh
-## @author Gichan Jang <gichan2.jang@samsung.com>
-## @date Mar 12 2021
-## @brief SSAT Test Cases for flexbuffers subplugin of tensor converter and decoder
-## @details After decoding the tensor into flexbuffer, convert it to tensor(s) again to check if it matches the original
-##
-if [[ "$SSATAPILOADED" != "1" ]]; then
-    SILENT=0
-    INDEPENDENT=1
-    search="ssat-api.sh"
-    source $search
-    printf "${Blue}Independent Mode${NC}"
-fi
-
-# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
-testInit $1
-
-if [ "$SKIPGEN" == "YES" ]; then
-    echo "Test Case Generation Skipped"
-    sopath=$2
-else
-    echo "Test Case Generation Started"
-    python ../nnstreamer_converter/generateGoldenTestResult.py 9
-    python3 ../nnstreamer_merge/generateTest.py
-    sopath=$1
-fi
-convertBMP2PNG
-
-PATH_TO_PLUGIN="../../build"
-
-##
-## @brief Execute gstreamer pipeline and compare the output of the pipeline
-## @param $1 Colorspace
-## @param $2 Width
-## @param $3 Height
-## @param $4 Test Case Number
-function do_test() {
-    gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=3 pattern=13 ! video/x-raw,format=${1},width=${2},height=${3},framerate=5/1 ! \
-    tee name=t ! queue ! multifilesink location=\"raw_${1}_${2}x${3}_%1d.log\"
-    t. ! queue ! tensor_converter ! tensor_decoder mode=flexbuf ! other/flexbuf ! tensor_converter ! multifilesink location=\"flexb_${1}_${2}x${3}_%1d.log\" sync=true" ${4} 0 0 $PERFORMANCE
-
-    callCompareTest raw_${1}_${2}x${3}_0.log flexb_${1}_${2}x${3}_0.log "${4}-1" "flexbuf conversion test ${4}-1" 1 0
-    callCompareTest raw_${1}_${2}x${3}_1.log flexb_${1}_${2}x${3}_1.log "${4}-2" "flexbuf conversion test ${4}-2" 1 0
-    callCompareTest raw_${1}_${2}x${3}_2.log flexb_${1}_${2}x${3}_2.log "${4}-3" "flexbuf conversion test ${4}-3" 1 0
-}
-# The width and height of video should be multiple of 4
-do_test BGRx 320 240 1-1
-do_test RGB 320 240 1-2
-do_test GRAY8 320 240 1-3
-
-# audio format S16LE, 8k sample rate, samples per buffer 8000
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=S16LE,rate=8000 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! tensor_decoder mode=flexbuf ! \
-        other/flexbuf ! tensor_converter ! filesink location=\"test.audio8k.s16le.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio8k.s16le.origin.log\" sync=true" 2-1 0 0 $PERFORMANCE
-callCompareTest test.audio8k.s16le.origin.log test.audio8k.s16le.log 2-2 "Audio8k-s16le Golden Test" 0 0
-
-# audio format U8, 16k sample rate, samples per buffer 8000
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=U8,rate=16000 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! tensor_decoder mode=flexbuf ! \
-        other/flexbuf ! tensor_converter ! filesink location=\"test.audio16k.u8.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio16k.u8.origin.log\" sync=true" 2-3 0 0 $PERFORMANCE
-callCompareTest test.audio16k.u8.origin.log test.audio16k.u8.log 2-4 "Audio16k-u8 Golden Test" 0 0
-
-# audio format U16LE, 16k sample rate, 2 channels, samples per buffer 8000
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=U16LE,rate=16000,channels=2 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! tensor_decoder mode=flexbuf ! \
-        other/flexbuf ! tensor_converter ! filesink location=\"test.audio16k2c.u16le.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio16k2c.u16le.origin.log\" sync=true" 2-5 0 0 $PERFORMANCE
-callCompareTest test.audio16k2c.u16le.origin.log test.audio16k2c.u16le.log 2-6 "Audio16k2c-u16le Golden Test" 0 0
-
-# Test other/tensors
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_mux name=tensors_mux sync-mode=basepad sync-option=1:50000000 ! \
-    tensor_decoder mode=flexbuf ! other/flexbuf ! tensor_converter ! multifilesink location=testsynch19_%1d.log \
-    tensor_mux name=tensor_mux0  sync-mode=slowest ! queue ! tensors_mux.sink_0 \
-    tensor_mux name=tensor_mux1  sync-mode=slowest ! queue ! tensors_mux.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)10/1\" ! pngdec ! tensor_converter ! tensor_mux0.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! tensor_converter ! tensor_mux0.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)30/1\" ! pngdec ! tensor_converter ! tensor_mux1.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! tensor_converter ! tensor_mux1.sink_1" 3 0 0 $PERFORMANCE
-callCompareTest testsynch19_0.golden testsynch19_0.log 3-1 "Tensor mux Compare 3-1" 1 0
-callCompareTest testsynch19_1.golden testsynch19_1.log 3-2 "Tensor mux Compare 3-2" 1 0
-callCompareTest testsynch19_2.golden testsynch19_2.log 3-3 "Tensor mux Compare 3-3" 1 0
-callCompareTest testsynch19_3.golden testsynch19_3.log 3-4 "Tensor mux Compare 3-4" 1 0
-callCompareTest testsynch19_4.golden testsynch19_4.log 3-5 "Tensor mux Compare 3-5" 1 0
-
-# Consecutive converting test
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=S16LE,rate=8000 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! \
-    tensor_decoder mode=flexbuf ! other/flexbuf ! tensor_converter ! \
-    tensor_decoder mode=flexbuf ! other/flexbuf ! tensor_converter ! \
-    tensor_decoder mode=flexbuf ! other/flexbuf ! tensor_converter ! filesink location=\"test.consecutive.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio8k.s16le.origin.log\" sync=true" 4 0 0 $PERFORMANCE
-callCompareTest test.audio8k.s16le.origin.log test.consecutive.log 4-1 "Consecutive converting test" 0 0
-
-rm *.log *.bmp *.png *.golden *.raw *.dat
-
-report
diff --git a/tests/nnstreamer_grpc/runTest.sh b/tests/nnstreamer_grpc/runTest.sh
deleted file mode 100755
index ed016d1f..00000000
--- a/tests/nnstreamer_grpc/runTest.sh
+++ /dev/null
@@ -1,154 +0,0 @@
-#!/usr/bin/env bash
-# SPDX-License-Identifier: LGPL-2.1-only
-#
-# @file runTest.sh
-# @author Dongju Chae <dongju.chae@samsung.com>
-# @date Nov 04 2020
-# @brief SSAT Test Cases for NNStreamer
-#
-
-if [[ "$SSATAPILOADED" != "1" ]]; then
-    SILENT=0
-    INDEPENDENT=1
-    search="ssat-api.sh"
-    source $search
-    printf "${Blue}Independent Mode${NC}"
-fi
-
-# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
-testInit $1
-
-# Skip test on qemu env.
-if [[ "$SKIP_QEMU_ARM_INCOMPATIBLE_TESTS" == "1" ]]; then
-  echo "Skip ssat tests on qemu/arm and qemu/arm64 environments."
-  report
-  exit
-fi
-
-# Check gRPC availability
-PATH_TO_PLUGIN=${NNSTREAMER_BUILD_ROOT_PATH}
-if [[ -d $PATH_TO_PLUGIN ]]; then
-    ini_path="${PATH_TO_PLUGIN}/ext/nnstreamer"
-    if [[ -d ${ini_path} ]]; then
-        check=$(ls ${ini_path} | grep nnstreamer-grpc.so)
-        if [[ ! $check ]]; then
-            echo "Cannot find nnstreamer-grpc shared lib"
-            report
-            exit
-        fi
-    else
-        echo "Cannot find ${ini_path}"
-    fi
-fi
-
-# Check IDL availability
-PATH_TO_PLUGIN_EXTRA=${PATH_TO_PLUGIN}/ext/nnstreamer/extra
-TEST_PROTOBUF=1
-TEST_FLATBUF=1
-if [[ -d $PATH_TO_PLUGIN_EXTRA ]]; then
-  ini_path="${PATH_TO_PLUGIN_EXTRA}"
-  if [[ -d ${ini_path} ]]; then
-    check=$(ls ${ini_path} | grep nnstreamer_grpc_protobuf.so)
-    if [[ ! $check ]]; then
-      echo "Cannot find nnstreamer_grpc_protobuf shared lib"
-      TEST_PROTOBUF=0
-    fi
-    check=$(ls ${ini_path} | grep nnstreamer_grpc_flatbuf.so)
-    if [[ ! $check ]]; then
-      echo "Cannot find nnstreamer_grpc_flatbuf shared lib"
-      TEST_FLATBUF=0
-    fi
-  else
-    echo "Cannot find ${ini_path}"
-  fi
-fi
-
-export LD_LIBRARY_PATH=${PATH_TO_PLUGIN_EXTRA}:${LD_LIBRARY_PATH}
-
-NUM_BUFFERS=10
-
-## Initial pipelines to generate reference outputs
-# passthrough, other/tensor
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=${NUM_BUFFERS} ! video/x-raw,width=640,height=480,framerate=5/1 ! tensor_converter ! multifilesink location=original1_%1d.log" Initial-1 0 0 $PERFORMANCE
-# passthrough, other/tensors
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=${NUM_BUFFERS} ! video/x-raw,width=640,height=480,framerate=5/1 ! tensor_converter frames-per-tensor=2 ! multifilesink location=original2_%1d.log" Initial-2 0 0 $PERFORMANCE
-
-INDEX=1
-BLOCKING_LIST=("TRUE" "FALSE")
-IDL_LIST=()
-if [[ $TEST_PROTOBUF -eq 1 ]]; then
-  IDL_LIST[${#IDL_LIST[@]}]="Protobuf"
-fi
-if [[ $TEST_FLATBUF -eq 1 ]]; then
-  IDL_LIST[${#IDL_LIST[@]}]="Flatbuf"
-fi
-
-## Test gRPC multi-pipelines with different IDL and blocking modes.
-for IDL in "${IDL_LIST[@]}"; do
-for BLOCKING in "${BLOCKING_LIST[@]}"; do
-  PORT=`python3 get_available_port.py`
-  BLOCKING_STR="Blocking"
-  if [[ $BLOCKING == "FALSE" ]]; then
-    BLOCKING_STR="Non-blocking"
-  fi
-
-  # tensor_sink (client) --> tensor_src (server), other/tensor
-  gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_src_grpc port=${PORT} num-buffers=${NUM_BUFFERS} idl=${IDL} blocking=${BLOCKING} ! 'other/tensor,dimension=(string)3:640:480,type=(string)uint8,framerate=(fraction)5/1' ! multifilesink location=result_%1d.log" ${INDEX}-1 0 0 $PERFORMANCE &
-  sleep 1
-  gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=${NUM_BUFFERS} ! video/x-raw,width=640,height=480,framerate=5/1 ! tensor_converter ! tensor_sink_grpc port=${PORT} idl=${IDL} blocking=${BLOCKING}" ${INDEX}-2 0 0 $PERFORMANCE
-
-  for i in `seq 0 $((NUM_BUFFERS-1))`
-  do
-    callCompareTest original1_${i}.log result_${i}.log GoldenTest-${INDEX} "gRPC ${IDL}/${BLOCKING_STR} $((i+1))/${NUM_BUFFERS}" 0 0
-  done
-
-  INDEX=$((INDEX + 1))
-  rm result_*.log
-
-  PORT=`python3 get_available_port.py`
-  # tensor_sink (server) --> tensor_src (client), other/tensor
-  gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=${NUM_BUFFERS} ! video/x-raw,width=640,height=480,framerate=5/1 ! tensor_converter ! tensor_sink_grpc port=${PORT} server=true idl=${IDL} blocking=${BLOCKING}" ${INDEX}-1 0 0 $PERFORMANCE &
-  sleep 1
-  gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_src_grpc port=${PORT} num-buffers=${NUM_BUFFERS} server=false idl=${IDL} blocking=${BLOCKING} ! 'other/tensor,dimension=(string)3:640:480,type=(string)uint8,framerate=(fraction)5/1' ! multifilesink location=result_%1d.log" ${INDEX}-2 0 0 $PERFORMANCE
-
-  for i in `seq 0 $((NUM_BUFFERS-1))`
-  do
-    callCompareTest original1_${i}.log result_${i}.log GoldenTest-${INDEX} "gRPC ${IDL}/${BLOCKING_STR} $((i+1))/${NUM_BUFFERS}" 0 0
-  done
-
-  INDEX=$((INDEX + 1))
-  rm result_*.log
-
-  PORT=`python3 get_available_port.py`
-  # tensor_sink (client) --> tensor_src (server), other/tensors
-  gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_src_grpc port=${PORT} num-buffers=$((NUM_BUFFERS/2)) idl=${IDL} blocking=${BLOCKING} ! 'other/tensors,num_tensors=2,dimensions=(string)3:640:480.3:640:480,types=(string)uint8.uint8,framerate=(fraction)5/1' ! multifilesink location=result_%1d.log" ${INDEX}-1 0 0 $PERFORMANCE &
-  sleep 1
-  gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=${NUM_BUFFERS} ! video/x-raw,width=640,height=480,framerate=5/1 ! tensor_converter frames-per-tensor=2 ! tensor_sink_grpc port=${PORT} idl=${IDL} blocking=${BLOCKING}" ${INDEX}-2 0 0 $PERFORMANCE
-
-  for i in `seq 0 $((NUM_BUFFERS/2-1))`
-  do
-    callCompareTest original2_${i}.log result_${i}.log GoldenTest-${INDEX} "gRPC ${IDL}/${BLOCKING_STR} $((i+1))/$((NUM_BUFFERS/2))" 0 0
-  done
-
-  INDEX=$((INDEX + 1))
-  rm result_*.log
-
-  PORT=`python3 get_available_port.py`
-  # tensor_sink (server) --> tensor_src (client), other/tensors
-  gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=${NUM_BUFFERS} ! video/x-raw,width=640,height=480,framerate=5/1 ! tensor_converter frames-per-tensor=2 ! tensor_sink_grpc port=${PORT} server=true idl=${IDL} blocking=${BLOCKING}" ${INDEX}-1 0 0 $PERFORMANCE &
-  sleep 1
-  gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_src_grpc port=${PORT} num-buffers=$((NUM_BUFFERS/2)) server=false idl=${IDL} blocking=${BLOCKING} ! 'other/tensors,num_tensors=2,dimensions=(string)3:640:480.3:640:480,types=(string)uint8.uint8,framerate=(fraction)5/1' ! multifilesink location=result_%1d.log" ${INDEX}-2 0 0 $PERFORMANCE
-
-  for i in `seq 0 $((NUM_BUFFERS/2-1))`
-  do
-    callCompareTest original2_${i}.log result_${i}.log GoldenTest-${INDEX} "gRPC ${IDL}/${BLOCKING_STR} $((i+1))/$((NUM_BUFFERS/2))" 0 0
-  done
-
-  INDEX=$((INDEX + 1))
-  rm result_*.log
-done
-done
-
-rm original*.log
-
-report
diff --git a/tests/nnstreamer_protobuf/runTest.sh b/tests/nnstreamer_protobuf/runTest.sh
deleted file mode 100755
index 286b3e56..00000000
--- a/tests/nnstreamer_protobuf/runTest.sh
+++ /dev/null
@@ -1,131 +0,0 @@
-#!/usr/bin/env bash
-##
-## SPDX-License-Identifier: LGPL-2.1-only
-##
-## @file runTest.sh
-## @author Gichan Jang <gichan2.jang@samsung.com>
-## @date June 08 2020
-## @brief SSAT Test Cases for protobuf subplugin of tensor converter and decoder
-## @details After decoding the tensor into protobuf, convert it to tensor(s) again to check if it matches the original
-##
-if [[ "$SSATAPILOADED" != "1" ]]; then
-    SILENT=0
-    INDEPENDENT=1
-    search="ssat-api.sh"
-    source $search
-    printf "${Blue}Independent Mode${NC}"
-fi
-
-# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
-testInit $1
-
-if [ "$SKIPGEN" == "YES" ]; then
-    echo "Test Case Generation Skipped"
-    sopath=$2
-else
-    echo "Test Case Generation Started"
-    python ../nnstreamer_converter/generateGoldenTestResult.py 9
-    python3 ../nnstreamer_merge/generateTest.py
-    sopath=$1
-fi
-convertBMP2PNG
-
-PATH_TO_PLUGIN="../../build"
-
-##
-## @brief Execute gstreamer pipeline and compare the output of the pipeline
-## @param $1 Colorspace
-## @param $2 Width
-## @param $3 Height
-## @param $4 Test Case Number
-function do_test() {
-    gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=3 pattern=13 ! video/x-raw,format=${1},width=${2},height=${3},framerate=5/1 ! \
-    tee name=t ! queue ! multifilesink location=\"raw_${1}_${2}x${3}_%1d.log\"
-    t. ! queue ! tensor_converter ! tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! multifilesink location=\"protob_${1}_${2}x${3}_%1d.log\" sync=true" ${4} 0 0 $PERFORMANCE
-
-    callCompareTest raw_${1}_${2}x${3}_0.log protob_${1}_${2}x${3}_0.log "${4}-1" "protobuf conversion test ${4}-1" 1 0
-    callCompareTest raw_${1}_${2}x${3}_1.log protob_${1}_${2}x${3}_1.log "${4}-2" "protobuf conversion test ${4}-2" 1 0
-    callCompareTest raw_${1}_${2}x${3}_2.log protob_${1}_${2}x${3}_2.log "${4}-3" "protobuf conversion test ${4}-3" 1 0
-}
-# The width and height of video should be multiple of 4
-do_test BGRx 320 240 1-1
-do_test RGB 320 240 1-2
-do_test GRAY8 320 240 1-3
-
-# audio format S16LE, 8k sample rate, samples per buffer 8000
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=S16LE,rate=8000 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! tensor_decoder mode=protobuf ! \
-        other/protobuf-tensor ! tensor_converter ! filesink location=\"test.audio8k.s16le.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio8k.s16le.origin.log\" sync=true" 2-1 0 0 $PERFORMANCE
-callCompareTest test.audio8k.s16le.origin.log test.audio8k.s16le.log 2-2 "Audio8k-s16le Golden Test" 0 0
-
-# audio format U8, 16k sample rate, samples per buffer 8000
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=U8,rate=16000 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! tensor_decoder mode=protobuf ! \
-        other/protobuf-tensor ! tensor_converter ! filesink location=\"test.audio16k.u8.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio16k.u8.origin.log\" sync=true" 2-3 0 0 $PERFORMANCE
-callCompareTest test.audio16k.u8.origin.log test.audio16k.u8.log 2-4 "Audio16k-u8 Golden Test" 0 0
-
-# audio format U16LE, 16k sample rate, 2 channels, samples per buffer 8000
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=U16LE,rate=16000,channels=2 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! tensor_decoder mode=protobuf ! \
-        other/protobuf-tensor ! tensor_converter ! filesink location=\"test.audio16k2c.u16le.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio16k2c.u16le.origin.log\" sync=true" 2-5 0 0 $PERFORMANCE
-callCompareTest test.audio16k2c.u16le.origin.log test.audio16k2c.u16le.log 2-6 "Audio16k2c-u16le Golden Test" 0 0
-
-# tensor merge test (The output is always in the format of other/tensor)
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN}  tensor_merge name=merge mode=linear option=2 silent=true sync-mode=basepad sync-option=0:0 ! multifilesink location=testsynch08_%1d.log \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)10/1\" ! pngdec ! tensor_converter !  \
-        tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! merge.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)30/1\" ! pngdec ! tensor_converter ! \
-        tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! merge.sink_1" 3 0 0 $PERFORMANCE
-callCompareTest testsynch08_0.golden testsynch08_0.log 3-1 "Tensor merge Compare 3-1" 1 0
-callCompareTest testsynch08_1.golden testsynch08_1.log 3-2 "Tensor merge Compare 3-2" 1 0
-callCompareTest testsynch08_2.golden testsynch08_2.log 3-3 "Tensor merge Compare 3-3" 1 0
-callCompareTest testsynch08_3.golden testsynch08_3.log 3-4 "Tensor merge Compare 3-4" 1 0
-
-# tensor mux test (The output is always in the format of other/tensors)
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_mux name=tensors_mux sync-mode=basepad sync-option=1:50000000 ! multifilesink location=testsynch19_%1d.log \
-    tensor_mux name=tensor_mux0  sync-mode=slowest ! tensors_mux.sink_0 \
-    tensor_mux name=tensor_mux1  sync-mode=slowest ! tensors_mux.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)10/1\" ! pngdec ! \
-        tensor_converter ! tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! tensor_mux0.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! \
-        tensor_converter ! tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! tensor_mux0.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)30/1\" ! pngdec ! \
-        tensor_converter ! tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! tensor_mux1.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! \
-        tensor_converter ! tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! tensor_mux1.sink_1" 4 0 0 $PERFORMANCE
-callCompareTest testsynch19_0.golden testsynch19_0.log 4-1 "Tensor mux Compare 4-1" 1 0
-callCompareTest testsynch19_1.golden testsynch19_1.log 4-2 "Tensor mux Compare 4-2" 1 0
-callCompareTest testsynch19_2.golden testsynch19_2.log 4-3 "Tensor mux Compare 4-3" 1 0
-callCompareTest testsynch19_3.golden testsynch19_3.log 4-4 "Tensor mux Compare 4-4" 1 0
-callCompareTest testsynch19_4.golden testsynch19_4.log 4-5 "Tensor mux Compare 4-5" 1 0
-
-# test other/tensors
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_mux name=tensors_mux sync-mode=basepad sync-option=1:50000000 ! \
-    tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! multifilesink location=testsynch19_5_%1d.log \
-    tensor_mux name=tensor_mux0  sync-mode=slowest ! tensors_mux.sink_0 \
-    tensor_mux name=tensor_mux1  sync-mode=slowest ! tensors_mux.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)10/1\" ! pngdec ! tensor_converter ! tensor_mux0.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! tensor_converter ! tensor_mux0.sink_1 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)30/1\" ! pngdec ! tensor_converter ! tensor_mux1.sink_0 \
-    multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! tensor_converter ! tensor_mux1.sink_1" 5 0 0 $PERFORMANCE
-callCompareTest testsynch19_0.golden testsynch19_5_0.log 5-1 "Tensor mux Compare 5-1" 1 0
-callCompareTest testsynch19_1.golden testsynch19_5_1.log 5-2 "Tensor mux Compare 5-2" 1 0
-callCompareTest testsynch19_2.golden testsynch19_5_2.log 5-3 "Tensor mux Compare 5-3" 1 0
-callCompareTest testsynch19_3.golden testsynch19_5_3.log 5-4 "Tensor mux Compare 5-4" 1 0
-callCompareTest testsynch19_4.golden testsynch19_5_4.log 5-5 "Tensor mux Compare 5-5" 1 0
-
-# Consecutive converting test
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesperbuffer=8000 ! audioconvert ! audio/x-raw,format=S16LE,rate=8000 ! \
-    tee name=t ! queue ! audioconvert ! tensor_converter frames-per-tensor=8000 ! \
-    tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! \
-    tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! \
-    tensor_decoder mode=protobuf ! other/protobuf-tensor ! tensor_converter ! filesink location=\"test.consecutive.log\" sync=true \
-    t. ! queue ! filesink location=\"test.audio8k.s16le.origin.log\" sync=true" 6 0 0 $PERFORMANCE
-callCompareTest test.audio8k.s16le.origin.log test.consecutive.log 6-1 "Consecutive converting test" 0 0
-
-rm *.log *.bmp *.png *.golden *.raw *.dat
-
-report
diff --git a/tests/nnstreamer_query/runTest.sh b/tests/nnstreamer_query/runTest.sh
deleted file mode 100644
index 5f47971a..00000000
--- a/tests/nnstreamer_query/runTest.sh
+++ /dev/null
@@ -1,52 +0,0 @@
-#!/usr/bin/env bash
-# SPDX-License-Identifier: LGPL-2.1-only
-#
-# @file runTest.sh
-# @author Gichan Jang <gichan2.jang@samsung.com>
-# @date Aug 25 2021
-# @brief SSAT Test Cases for tensor query
-#
-if [[ "$SSATAPILOADED" != "1" ]]; then
-    SILENT=0
-    INDEPENDENT=1
-    search="ssat-api.sh"
-    source $search
-    printf "${Blue}Independent Mode${NC}
-"
-fi
-
-# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
-testInit $1
-
-PATH_TO_PLUGIN="../../build"
-PERFORMANCE=0
-TIMEOUT_SEC=5
-
-# Run tensor query server as echo server with default adress option.
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_query_serversrc num-buffers=3 ! other/tensors,num_tensors=1,dimensions=3:300:300:1,types=uint8 ! tensor_query_serversink" 1-1 0 0 $PERFORMANCE $TIMEOUT_SEC &
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=3 ! videoconvert ! videoscale ! video/x-raw,width=300,height=300,format=RGB ! tensor_converter ! tee name = t t. ! queue ! multifilesink location= raw1_%1d.log t. ! queue ! tensor_query_client ! multifilesink location=result1_%1d.log" 1-2 0 0 $PERFORMANCE
-callCompareTest raw1_0.log result1_0.log 1-3 "Compare 1-3" 1 0
-callCompareTest raw1_1.log result1_1.log 1-4 "Compare 1-4" 1 0
-callCompareTest raw1_2.log result1_2.log 1-5 "Compare 1-5" 1 0
-
-# Run tensor query server as echo server with given address option. (multi clients)
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_query_serversrc host=127.0.0.1 port=5001 num-buffers=6 ! other/tensors,num_tensors=1,dimensions=3:300:300:1,types=uint8 ! tensor_query_serversink host=127.0.0.1 port=5002" 2-1 0 0 $PERFORMANCE $TIMEOUT_SEC &
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=3 ! videoconvert ! videoscale ! video/x-raw,width=300,height=300,format=RGB ! tensor_converter ! tee name = t t. ! queue ! multifilesink location= raw2_%1d.log t. ! queue ! tensor_query_client src-host=127.0.0.1 src-port=5001 sink-host=127.0.0.1 sink-port=5002 ! multifilesink location=result2_%1d.log" 2-2 0 0 $PERFORMANCE &
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=3 ! videoconvert ! videoscale ! video/x-raw,width=300,height=300,format=RGB ! tensor_converter ! tee name = t t. ! queue ! multifilesink location= raw2_2_%1d.log t. ! queue ! tensor_query_client src-host=127.0.0.1 src-port=5001 sink-host=127.0.0.1 sink-port=5002 ! multifilesink location=result2_2_%1d.log" 2-3 0 0 $PERFORMANCE
-callCompareTest raw2_0.log result2_0.log 2-4 "Compare 2-4" 1 0
-callCompareTest raw2_1.log result2_1.log 2-5 "Compare 2-5" 1 0
-callCompareTest raw2_2.log result2_2.log 2-6 "Compare 2-6" 1 0
-callCompareTest raw2_2_0.log result2_2_0.log 2-7 "Compare 2-7" 1 0
-callCompareTest raw2_2_1.log result2_2_1.log 2-8 "Compare 2-8" 1 0
-callCompareTest raw2_2_2.log result2_2_2.log 2-9 "Compare 2-9" 1 0
-
-# Test flexible tensors
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} tensor_query_serversrc num-buffers=3 ! other/tensors,format=flexible ! tensor_query_serversink" 3-1 0 0 $PERFORMANCE $TIMEOUT_SEC &
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=3 ! videoconvert ! videoscale ! video/x-raw,width=300,height=300,format=RGB ! tensor_converter ! other/tensors,format=flexible ! tee name = t t. ! queue ! multifilesink location= raw3_%1d.log t. ! queue ! tensor_query_client ! multifilesink location=result3_%1d.log" 3-2 0 0 $PERFORMANCE
-callCompareTest raw3_0.log result3_0.log 3-3 "Compare 3-3" 1 0
-callCompareTest raw3_1.log result3_1.log 3-4 "Compare 3-4" 1 0
-callCompareTest raw3_2.log result3_2.log 3-5 "Compare 3-5" 1 0
-
-rm *.log
-
-report
diff --git a/tests/nnstreamer_repo_dynamicity/runTest.sh b/tests/nnstreamer_repo_dynamicity/runTest.sh
deleted file mode 100644
index 84657ab5..00000000
--- a/tests/nnstreamer_repo_dynamicity/runTest.sh
+++ /dev/null
@@ -1,57 +0,0 @@
-#!/usr/bin/env bash
-##
-## SPDX-License-Identifier: LGPL-2.1-only
-##
-## @file runTest.sh
-## @author MyungJoo Ham <myungjoo.ham@gmail.com>
-## @date Nov 01 2018
-## @brief SSAT Test Cases for NNStreamer
-##
-if [[ "$SSATAPILOADED" != "1" ]]; then
-    SILENT=0
-    INDEPENDENT=1
-    search="ssat-api.sh"
-    source $search
-    printf "${Blue}Independent Mode${NC}
-"
-fi
-
-# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
-testInit $1
-
-PATH_TO_PLUGIN="../../build"
-
-if [ "$SKIPGEN" == "YES" ]; then
-    echo "Test Case Generation Skipped"
-    sopath=$2
-else
-    echo "Test Case Generation Started"
-    python ../nnstreamer_converter/generateGoldenTestResult.py 12
-    sopath=$1
-fi
-convertBMP2PNG
-
-if [[ ! -z "${UNITTEST_DIR}" ]]; then
-    TESTBINDIR="${UNITTEST_DIR}"
-elif [ ! -d "${PATH_TO_PLUGIN}" ] && [ ! -d "${UNITTEST_DIR}" ]; then
-    TESTBINDIR="/usr/bin/unittest-nnstreamer/tests"
-else
-    TESTBINDIR="../../build/tests/nnstreamer_repo_dynamicity"
-fi
-
-${TESTBINDIR}/unittest_repo --gst-plugin-path=../../build
-
-callCompareTest testsequence_1.golden tensorsequence01_1.log 1-1 "Compare 1-1" 1 0
-callCompareTest testsequence_2.golden tensorsequence01_2.log 1-2 "Compare 1-2" 1 0
-callCompareTest testsequence_3.golden tensorsequence01_3.log 1-3 "Compare 1-3" 1 0
-callCompareTest testsequence_4.golden tensorsequence01_4.log 1-4 "Compare 1-4" 1 0
-callCompareTest testsequence_5.golden tensorsequence01_5.log 1-5 "Compare 1-5" 1 0
-callCompareTest testsequence_6.golden tensorsequence01_6.log 1-6 "Compare 1-6" 1 0
-callCompareTest testsequence_7.golden tensorsequence01_7.log 1-7 "Compare 1-7" 1 0
-callCompareTest testsequence_8.golden tensorsequence01_8.log 1-8 "Compare 1-8" 1 0
-callCompareTest testsequence_9.golden tensorsequence01_9.log 1-9 "Compare 1-9" 1 0
-callCompareTest testsequence_10.golden tensorsequence01_10.log 1-10 "Compare 1-10" 1 0
-
-rm *.log *.bmp *.png *.golden *.raw *.dat
-
-report
-- 
2.17.1

